tools/azure-devtools/src/azure_devtools/scenario_tests/preparers.py

```diff
@@ -135,7 +135,9 @@ def _preparer_wrapper(test_class_instance, **kwargs):
                         )
 
             if test_class_instance.is_live:
-                test_class_instance.scrubber.register_name_pair(resource_name, self.moniker)
+                # Adding this for new proxy testcase
+                if hasattr(test_class_instance, "scrubber"):
+                    test_class_instance.scrubber.register_name_pair(resource_name, self.moniker)
```

Comment:
Nit: I would add a log as an else statement here because then the name pair will not be scrubbed from the recordings. With a log at least you are notifying the dev that the object has no scrubber object.